pandas/core/dtypes/common.py

```diff
@@ -1905,20 +1906,20 @@ def _get_dtype_type(arr_or_dtype):
     elif isinstance(arr_or_dtype, CategoricalDtype):
         return CategoricalDtypeType
```

Comment:
Did you seem my comment at https://github.com/pandas-dev/pandas/pull/22938#discussion_r221993329 about categorical? We posted at the same time.

We discussed changing it to `object` earlier, but didn't see the benefit. Categorical doesn't really have a specific "type", so it would have to be super-general, which makes `.type` kind of useless.